tests/e2e/tests/checkout/checkout-block-place-order.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { CheckoutPage } from './checkout.page';
+import {
+	SIMPLE_PHYSICAL_PRODUCT_NAME,
+	FREE_SHIPPING_NAME,
+	FREE_SHIPPING_PRICE,
+} from './constants';
+
+const test = base.extend< { pageObject: CheckoutPage } >( {
+	pageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Checkout block → Place Order → customer', () => {
+	test.use( { storageState: 'playwright/.auth/customer.json' } );
+
+	test( 'Guest user can place order', async ( {
```

Comment:
Do you think this test case is covered by https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tests/e2e/tests/checkout/checkout-block-shipping.block_theme.side_effects.spec.ts#L74-L81 instead? If we execute `test.use( { storageState: 'playwright/.auth/customer.json' } );` in that file, we're already testing if we can place the order in that file so probably don't need a separate test for it?